pinot-core/src/main/java/org/apache/pinot/core/io/reader/impl/v1/FixedBitMultiValueReader.java

```diff
@@ -49,8 +51,8 @@
   private static final int PREFERRED_NUM_VALUES_PER_CHUNK = 2048;
 
   private final FixedByteValueReaderWriter _chunkOffsetReader;
-  private final PinotDataBitSet _bitmapReader;
-  private final FixedBitIntReaderWriter _rawDataReader;
+  private final PinotDataBuffer _headerBitmapBuffer;
+  private final FixedBitIntReaderWriterV2 _rawDataReader;
```

Comment:
I would suggest removing `FixedBitIntReaderWriter` and `FixedBitIntReaderWriterV2` and directly use `PinotBitSet`. There is no value added to this extra layer.